components/map/events.md

```diff
@@ -304,7 +321,223 @@ The `OnShapeClick` event fires when the user clicks on a shape. Its `EventCallba
             $"clientX = {eventArgs.ClientX}, clientY = {eventArgs.ClientY}");
     }
 
-    public void LogToConsole(string text)
+    private void LogToConsole(string text)
+    {
+        EventResult = text;
+    }
+
+    public class MarkerModel
+    {
+        public double[] LatLng { get; set; }
+        public string Title { get; set; }
+    }
+
+    public class BubbleModel
+    {
+        public double[] LatLng { get; set; }
+        public int Revenue { get; set; }
+    }
+}
+````
+
+## OnZoomEnd 
+
+The `OnZoomEnd` event fires when the user has finished zooming the Map. The `OnZoomEnd` event handler receives an argument of type `MapZoomEndEventArgs`, which exposes the following properties:
```

Comment:
Double-check your use of map vs Map here, in onPanEnd and in the OnClick section. Apply consistency as needed.